orderer/common/admin/system.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package admin
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/hyperledger/fabric/common/flogging"
+	"github.com/hyperledger/fabric/common/util"
+	"github.com/hyperledger/fabric/core/middleware"
+)
+
+//go:generate counterfeiter -o fakes/logger.go -fake-name Logger . Logger
+
+type Logger interface {
+	Warn(args ...interface{})
+	Warnf(template string, args ...interface{})
+}
+
+type Options struct {
+	Logger        Logger
+	ListenAddress string
+	TLS           TLS
+	Version       string
+}
+
+type System struct {
```

Comment:
Wasn't the whole point of factoring the operations.System out into system.System so that you could re-use it? This seems like entirely duplicated code?

Reply:
Hmmm, I'm honestly not sure what happened with the code here. It looks like I messed things up when I split things into two commits. I'll get this back to what I intended and push an update...

Reply:
Oh right, this was my first pass making sure I understood the common code that was needed before splitting it out into a separate package. I'll remove this dead code.

Reply:
Removed.